### DISCUSSION TITLE
Need input on ENV data required while setting up Project locally by cloning

### DISCUSSION THREAD

**Discussion by PPnaruto** [rainbow-me/rainbowkit#2139] [Q&A] [answered]
Dear Team,

I hope this message finds you well.

As I am in the process of setting up the RainbowKit project locally to raise a PR, I have encountered an issue while installing pnpm. Specifically, I am receiving the following error:
"missing RAINBOW_PROVIDER_API_KEY env variable".

Upon reviewing the documentation, I understand that this environment variable needs to be configured on our end. Could you please assist me in obtaining the Rainbow Provider API key?

Additionally, I would require the values for the following environment variables to complete the setup:

> NEXT_PUBLIC_ALCHEMY_ID= # Required for Alchemy provider
> NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID= # Required for WalletConnect v2
> NEXTAUTH_SECRET= # Mac/Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32

Your prompt assistance in providing these details would be greatly appreciated, as it will enable us to proceed with the PR as soon as possible.

Thank you for your support.

Best regards,
Pratik Patil

**Comment by DanielSinclair** [comment, marked as answer]
The `RAINBOW_PROVIDER_API_KEY` env variable is no longer required. Give it another try! You can generate a WalletConnect project ID here: https://cloud.reown.com/
